Makefile

```diff
@@ -4,15 +4,63 @@
 
 export $(shell sed 's/=.*//' ./.env)
 
+# Project
+export PROJECT := $(or $(PROJECT),ehpr)
+
+# Runtime and application Environments specific variable
 export NODE_ENV ?= development
+export ENV_NAME ?= dev
+
+# AWS Environments variables
+export AWS_REGION ?= ca-central-1
+NAMESPACE = $(PROJECT)-$(ENV_NAME)
+APP_SRC_BUCKET = $(NAMESPACE)-app
+
+# Terraform variables
+TERRAFORM_DIR = terraform
+export BOOTSTRAP_ENV=terraform/bootstrap
+
+define TFVARS_DATA
+target_env = "$(ENV_NAME)"
+project_code = "$(PROJECT)"
+api_artifact = "build/api.zip"
+app_sources = "build/app"
+app_sources_bucket = "$(APP_SRC_BUCKET)"
+domain = ""
+endef
+export TFVARS_DATA
+CLOUDFRONT_ID = EH5H3TN595NCP
```

Comment:
Is this something we should put into environment variables?

Reply:
Yes, I'm changing

Reply:
DONE